src/components/Flex.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <div :style="flexStyle">
+    <slot>
+      <h1>{{ label }}1</h1>
```

Comment:
we can remove this two <h1> tags, it's enough to be a <slot> tag. The slot element is left empty, allowing the parent component to add any kind of content it wants.